packages/metal-quartz-button/src/MetalQuartzButtonAttributes.soy

```diff
@@ -0,0 +1,36 @@
+{namespace MetalQuartzButtonAttributes}
+
+/**
+ * This renders the component's whole content.
```

Comment:
Maybe change to something like "Renders html attributes for MetalQuartzButton component"